src/content/docs/configuration/data-types.mdx

```diff
@@ -171,21 +171,21 @@ XXXX-XX-31T14:00/XXXX-XX-31T19:00[Europe/Paris]
 ## Relative Timestamp
 
 Timestamps can be expressed relative to the current date and time. The format
-is `[DD days] [HH:MM] ago` :
+is `[DD days] [HH hours] [MM minutes] ago` :
```

Comment:
Should we document also the short format? Like `8 h 30 min ago`.

Reply:
I prefer to keep it simple :)